### ISSUE TITLE
Certain Big5 characters can only be triggered through #show

### ISSUE THREAD

**Issue by rarealphacat** [scandum/tintin#180]
![Screenshot_2023-04-08_21-49-39](https://user-images.githubusercontent.com/28812960/230724776-15363757-8c5b-41fa-a925-8fb3b4f49bca.jpg)

This is very strange to me, some characters cannot be triggered if it comes from the game but it can be triggered through #show command.

**Comment by scandum** [comment]
Only one action can trigger at a time. Check with `#debug action on` to see if that's the case here?

**Comment by rarealphacat** [comment]
It is the only action in the session.

**Comment by scandum** [comment]
Odd.

Difficult for me to fix as I can't easily reproduce it. Does it trigger if you use #show with all 4 Big5 characters echoed by say?

Also, are you using BIG5 as the charset, or BIG5TOUTF8 ?

**Comment by rarealphacat** [comment]
I'm not sure I get your question, there is only problematic Big5 character I found now "不" and what do you mean by echoed by say, I thought #echo does not trigger at all? 
Do you mean to "#action {不} {#echo triggered}", then say 不? If so, no, it doesn't trigger.

Or do you mean [this?](https://imgur.com/NOu2i4k)
It still didn't trigger.

I am using BIG5TOUTF8.

**Comment by scandum** [comment]
I meant trying:

![image](https://user-images.githubusercontent.com/52607428/231225484-eb583678-a1f2-4c71-b7d7-d1c077acc832.png)

I suspect it's something related to pcre, which would be hard to fix.

**Comment by rarealphacat** [comment]
yes, it [triggered](https://imgur.com/a/aPJieGZ) and [this](https://imgur.com/LU3UNxm)
